src/components/EntityPage/EntityScorecardsCard.tsx

```diff
@@ -75,9 +75,37 @@ export const EntityScorecardsCard = ({
 }: EntityScorecardsCardProps) => {
   const classes = useStyles();
 
-  const [sortBy, setSortBy] = useDropdown('Name ↑');
   const [searchQuery, setSearchQuery] = useInput();
 
+  const {
+    compareFn: scorecardCompareFn,
+    loading: loadingScorecardCompareFn,
+    error: scorecardComapreFnError
+  } = usePartialScorecardCompareFn();
+
+  const customScoresSortMethods: SortMethods<ServiceScorecardScore> = useMemo(() => {
+    if (isUndefined(scorecardCompareFn)) {
+      return {} as SortMethods<ServiceScorecardScore>;
+    }
+
+    return {
+      'Custom ↑': (a: ServiceScorecardScore, b: ServiceScorecardScore) => scorecardCompareFn(a.scorecard, b.scorecard),
```

Comment:
[Non-blocking] I wonder if it makes sense to allow users to customize the name of the sort for the label here.

Reply:
would def be useful! honestly predicting this to be a pretty niche usecase but there's room to add a label to the existing extension API object